Resources/public/js/views/ez-navigationhubview.js

```diff
@@ -244,15 +241,29 @@ YUI.add('ez-navigationhubview', function (Y) {
             var container = this.get('container');
 
             container.setHTML(this.template({
-                user: this.get('user').toJSON(),
                 zones: this._buildZones(),
             }));
             this._renderNavigationItems();
             this._uiSetActiveNavigation();
+            this._renderUserProfile();
             return this;
         },
 
         /**
+         * Renders user profile to the view
+         *
+         * @method _renderUserProfile
+         * @protected
+         */
+        _renderUserProfile: function () {
+            var profileView = this.get('userProfileView');
+
+            profileView.addTarget(this);
+            this.get('container').append(profileView.render().get('container'));
+            profileView.set('active', true);
```

Comment:
that's not needed normally thanks to https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/public/js/views/ez-view.js#L53

(and kind of wrong, `active` means *attached to the DOM* which is not the case yet when this code is executed)